guides/v2.1/security/two-factor-authentication.md

```diff
@@ -0,0 +1,109 @@
+php bin/magento setup:upgrade
+```
+
+## Configure 2FA
+
+To enable and configure 2FA, see the [Magento Admin User Guide](https://docs.magento.com/m2/2.1/ee/user_guide/stores/security-two-factor-authentication.html).
+
+## Install authenticator
+
+After enabling and configuring 2FA, your staff will need to:
+
+1.	Install one of the supported authenticators to a mobile device or obtain a configured device.
+
+    For example, you can install the Google Authenticator app to a mobile device such as a smart phone or tablet. Depending on the OS, you can download and install the authenticator from [Google Play](https://support.google.com/accounts/answer/1066447?co=GENIE.Platform%3DAndroid&hl=en) or [iOS App Store](https://support.google.com/accounts/answer/1066447?co=GENIE.Platform%3DiOS&hl=en&oco=0).
+
+1.	Log into the Magento Admin and configure their authenticator. The first login prompts to configure the authenticator. For complete instructions, see [Using Two-Factor Authentication](https://docs.magento.com/m2/2.1/ee/user_guide/stores/security-two-factor-authentication-use.html).
+
+    For example, Google Authenticators will provide a QR code to configure access at Magento Admin login.
+
+Additional logins to the Admin from new devices will require the entered code or attached device.
+
+### Supported authenticators
+
+| Provider | Authentication Type |
+| --------- : | --------- : |
+| [Google Authenticator](https://support.google.com/accounts/answer/1066447?co=GENIE.Platform%3DAndroid&hl=en)  | Generate and enter code from mobile app Requirements: Enable in Admin  |
+| [Authy](https://authy.com/)  | SMS, call, token, and one touch <br/>Requirements: Enable and API keys  |
+| [U2F Keys](https://docs.magento.com/m2/2.1/ee/user_guide/stores/security-two-factor-authentication-use.html)   | Physical device to authenticate, [YubiKey](https://www.yubico.com/) and other models supported. <br/>Requirements: Enable in Admin  |
+| [Duo Security](https://duo.com/)  | SMS and push notification. <br/>Requirements: Enable, Integration and Secret keys, API hostname  |
+
+## Manage authenticators
+
+Admins have options for resetting authenticators, revoking access for devices, and troubleshooting issues through the [Magento Admin](https://docs.magento.com/m2/2.1/ee/user_guide/stores/security-two-factor-authentication-manage.html).
+
+## Troubleshooting
+
+The extension supports command line options for disabling, revoking, and resetting authenticators.
+
+### Disable authenticator
+
+If you have issues with 2FA, you can disable it from command-line. This will disable 2FA globally.
+
+``` bash
+php bin/magento msp:security:tfa:disable
+```
+
+### Reset authenticator per account
+
+If you need to manually reset a single user configuration, enter the following from the command-line. The command restarts configuration and 2FA subscription for the user account.
+
+``` bash
+php bin/magento msp:security:tfa:reset <username> <provider>
+```
+
```

Comment:
The table above needs to have a column that lists the <provider> name to be used for the command line. Then, you'll only need to provide one example reset command

Reply:
Adding that, thanks